### DISCUSSION TITLE
[SVG's with Gatsby] - Gatsby build process removing inline styles from SVG's

### DISCUSSION THREAD

**Discussion by hammund0** [gatsbyjs/gatsby#36047] [Help]
Hello,

I have added an inline style to an SVG but when I run gatsby build, it removes the animation. It works locally on develop but not when you build. 

I have spent hours googling to try to keep the animations in the SVG's, adding plugins etc, but to no avail.

Please can someone help, all i need to do is somehow get the image transforms to ignore the SVG's within a certain directory.

Assistance would be much appreciated. Thanks

**Comment by ngocdaothanh** [comment]
@hammund0 Try using the solution "add this to your gatsby-node.js" in https://github.com/gatsbyjs/gatsby/issues/31878, and try removing SVGO options like `removeStyleElement`.

I wonder if there's a more elegant/shorter way to remove only a list of SVGO options.
